projects/developer/src/app/navbar/navbar.component.ts

```diff
@@ -53,6 +56,10 @@ export class NavbarComponent implements OnInit, OnChanges, OnDestroy {
         });
     }
 
+    noBanner() {

```

Comment:
not a huge deal, but i'd prefer this to be an action-y type of function name, like "closeBanner" or "toggleBanner" or something.

Reply:
Makes sense, sure.